airgun/entities/ldap_authentication.py

```diff
@@ -53,6 +53,13 @@ def delete(self, entity_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def testConnection(self, values):
```

Comment:
keep the function name lowercase as test_connection or check_connection

Reply:
Requested changes made.